maas/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+
+  region-controller:
+    container_name: maas-region-controller
+    #env_file: ./region/.env
+    build: 
+      context: ./maas-region-controller
+    privileged: true
+    ports:
```

Comment:
This should also expose port 80 for the web UI and API (same REST endpoint for both)